### ISSUE TITLE
1.5.4 animation sluggish

### ISSUE THREAD

**Issue by humantorch** [binaryage/totalterminal#111]
Since the 1.5.4 update change to using animation groups, the animation has become very sluggish and choppy. I've got both slide and fade enabled, previous versions still ran both very smoothly but the frame rate has dropped considerably. Obviously not a huge issue as performance within the app is still the same as it ever was but something worth mentioning. 

MacBook Pro Retina 13", Late 2013
2,6 GHz i5
8 GB memory
Intel Iris 1536 MB 

Any other info you want, just let me know.

**Comment by darwin** [comment]
Yes, I also noticed it. I was so happy with the new implementation using animation groups. I will have to revert it to the old-dirty way probably.
